### PR TITLE
Fix Bruggeman EMA

### PR DESCRIPTION
**Pull request by MarJMue**
The selection of the correct root in the Jansson algorithm was faulty. 

This PR fixes one bug in the algorithm, but some numerical problems persist.
Additionally, a new method got implemented which seems to be more stable.

ToDo:
- [ ] Find out why the Jansson algorithm still fails on some samples.
- [x] Write more tests for mixture materials
- [x] Determine if the second method is a better default.

### CODE REVIEW

**Comment by MarJMue** [comment]
I have found the mistake for the new algorithm, and it seems to work in all cases I have tested.
(TIL sqrt(a/b) =/= sqrt(a) / sqrt(b) for complex numbers... Should have known better...)

I have subsequently disabled the old code path, as I could not find the error until now.

We might consider disabling the mixture tests or marking them as slow, as they take some time to finish.

File: src/elli/materials.py [hunk 1]
```diff
@@ -453,106 +453,157 @@ def get_tensor_fraction(self, lbda: npt.ArrayLike, fraction: float) -> npt.NDArr
...
-        correct_root = np.full_like(e_h, np.nan)
-        correct_root[mask_equal] = e_h[mask_equal]
-        correct_root[mask_straight] = check_straight_line()
-        correct_root[mask_general] = check_general_case()
-
-        return correct_root
+        return e_mix
+
+
+# class BruggemanJanssonEMA(MixtureMaterial):
```
Comment (domna) [inline src/elli/materials.py]: I would just remove it. It still can be recovered from git history

File: tests/test_mixture_models.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    material_list,
+)
+@pytest.mark.parametrize(
+    "EMA",
+    [
+        # elli.VCAMaterial,
+        elli.LooyengaEMA,
+        elli.MaxwellGarnettEMA,
+        # elli.BruggemanJanssonEMA,
+    ],
```
Comment (domna) [inline tests/test_mixture_models.py]:
```suggestion
    [
        elli.LooyengaEMA,
        elli.MaxwellGarnettEMA,
    ],
```

**Review by domna — APPROVED** [review]
I would remove the comments and re-add them if we want to activate them in the future. I did not fully check the algorithm but I think you did enough successful checking already.

**Pull request merged**